### DISCUSSION TITLE
Acerca de la estructura del proyecto.

### DISCUSSION THREAD

**Discussion by DarielF** [cl8dep/depingaasere-wiki#2] [Ideas]
## Sugerencia para la estructura
Actualmente se tiene la estructura de Home --> Guias --> {Panamá, Uruguay}
Para todo lo referente a migración sería mejor que en vez de Guía se usara _Migración_  lo cual encapsularía todo lo referente a este tema, dentro de _Migración_ vendrían los países y dentro de ellos la respectiva página con los distintos visados y estatus migratorios

Además de _Migración_  también deberían estar los siguientes temas:

- Negocios
- Ocio
- Otros

### Desloze de temas:
- **Negocios:** Todo lo relacionado a legislación para negocios, mercado de divisas fiat, criptomonedas, pasarelas de pago, lugares de compra, lugares para buscar trabajo, paquetería, etc.
- **Ocio:**  Lugares y rankings para el ocio, precio de bares, restaurantes, etc.
- **Otros:** Otros

**Comment by cl8dep** [comment]
Estoy de acuerdo. Go ahead with this changes
